backend/src/auth.middleware.ts

```diff
@@ -0,0 +1,29 @@
+import { Request, Response, NextFunction, response } from "express";
+import * as jwt from "jsonwebtoken";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+export default async function authUser(
+  req: Request,
+  res: Response,
+  next: NextFunction
+) {
+  try {
+    const cookie = req.headers.cookie;
```

Comment:
isn't it better if we make it in a header authentication bearer attribute rather than the cookie ?